From d1fb013bf055e3522fdd3e67c355e606a08d7611 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Fri, 24 Feb 2012 23:19:49 +0100
Subject: [PATCH 5/5] Revert "build: fixed build issue in certain cases"

The double include breaks all kinds of undefs, on ARM HAS_GL gets enabled

This reverts commit bf9dd471187f56eeb35a6f5ce4a3c39faaa7e72e.
---
 xbmc/cores/DllLoader/LibraryLoader.h |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/xbmc/cores/DllLoader/LibraryLoader.h b/xbmc/cores/DllLoader/LibraryLoader.h
index 7f5129f..a3f6c5d 100644
--- a/xbmc/cores/DllLoader/LibraryLoader.h
+++ b/xbmc/cores/DllLoader/LibraryLoader.h
@@ -22,8 +22,9 @@
 #ifndef LIBRARY_LOADER
 #define LIBRARY_LOADER
 
-#include "system.h"
-#ifdef _LINUX
+#ifndef _LINUX
+#include "system.h" // WIN32INCLUDES - needed for HMODULE
+#else
 #include "PlatformDefs.h"
 #endif
 
-- 
1.7.2.5

